src/Auth0Client/Auth0Client.cs

```diff
@@ -303,32 +302,31 @@ private static bool ScopeHasOfflineAccess(string scope)
 				.Any(e => e.Equals("offline_access", StringComparison.InvariantCultureIgnoreCase));
 		}
 
-		protected void SetupCurrentUser(IDictionary<string, string> accountProperties)
+		protected async Task SetupCurrentUser(IDictionary<string, string> accountProperties)
```

Comment:
Per .NET guidelines, this should be SetupCurrentUserAsync. Same w/ all the others. What do you think?